Deepgram/Projects/IProjectClient.cs

```diff
@@ -31,5 +31,12 @@ public interface IProjectClient
         /// </summary>
         /// <param name="projectId">Unique identifier of the project to delete</param>
         Task<MessageResponse> DeleteProjectAsync(string projectId);
+
+        /// <summary>
+        /// Returns all members of a project
+        /// </summary>
+        /// <param name="projectId"></param>

```

Comment:
Can we add a comment on this parameter `Unique identifier of the project for which you want to get members.`

Reply:
Yeah, sure. It's updated now.